proto/asn1/asn1-universal-types/asn1_universal_types_to_der.h

```diff
@@ -0,0 +1,33 @@
+#ifndef ASN1_UNIVERSAL_TYPES_TO_DER_H_
+#define ASN1_UNIVERSAL_TYPES_TO_DER_H_
+
+#include <vector>
```

Comment:
You'll also need stdint.h

We want to "include what you use": if a header uses a type, it should make sure to include the relevant header. Since you use uint8_t, you'll want that header here